test/base.py

```diff
@@ -45,33 +48,36 @@ def flatten(l, ltypes=(list, tuple)):
         i += 1
     return ltype(l)
 
+
 class ColumnName(str):
     pass
 
+
 def dbparam(x):
     if x is None:
         return 'NULL'
     elif isinstance(x, ColumnName):
         return "`" + x + "`"
-    elif isinstance(x, basestring):
+    elif isinstance(x, bytes):
+        return "'" + str(x) + "'"
+    elif isinstance(x, str):
```

Comment:
So at the top have a tuple with the string types:
```python
try:
   stringTypes = (str, unicode)
exception TypeError:
   stringTypes = (str,)
```
and then do `isinstance(x, stringTypes)`. (I may have the exception type wrong).